From 42033830e11235aa2608a926ae439fe4df48ae3e Mon Sep 17 00:00:00 2001
From: dmeier <dmeier@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 24 Aug 2016 09:01:00 +0000
Subject: [PATCH 0785/1076] MANTIS 40178 new component/devicehandler for
 TTS(text to speech)

Added parameter RC to playComplete() in AnnounceDev.idl

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1074 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/AnnounceDev.idl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/AnnounceDev.idl b/SelfServiceCommon/Massai/idl/Massai/AnnounceDev.idl
index 1c33bea5..4872ea60 100644
--- a/SelfServiceCommon/Massai/idl/Massai/AnnounceDev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/AnnounceDev.idl
@@ -32,11 +32,12 @@ module AnnounceDev
 
   interface AnnounceListener
   {
-    void playComplete();                /**< play is complete, i.e. speaking finished or stopped */
+    void playComplete(in RC rc);        /**< play is complete, i.e. speaking finished or stopped */
   };
 	
   interface AnnounceControl
   {
+    boolean is_connected(  );           /**< Always answers true, will timeout if not connected */
     RC play( in bytestream ssml );      /**< Play aka speak the text given in ssml */
     RC stop(  );                        /**< Stop playing text, playComplete() will be notified, does not work if paused */
 
@@ -45,7 +46,6 @@ module AnnounceDev
 
 		void setListener(in AnnounceListener listener);
 		void rmvListener(in AnnounceListener listener);    
-
 	};
 
 };
-- 
2.41.0.windows.1

